src/components/DatePicker/ModeSelector/DatePickerModeSelector.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Toggle } from '../../Toggle'
+import { ToggleSize } from '../../Toggle/Toggle'
+
+type SingularPickerMode = 'dayPicker' | 'monthPicker' | 'timePicker'
+type RangePickerMode = 'dayRangePicker' | 'monthRangePicker'
+
+export type DatePickerMode = SingularPickerMode | RangePickerMode
+
+const DATE_RANGE_MODE_CONFIG: Record<DatePickerMode, { label: string }> = {
+  timePicker: { label: 'Time' },
+  dayPicker: { label: 'Day' },
+  dayRangePicker: { label: 'Custom Date Range' },
```

Comment:
```suggestion
  dayRangePicker: { label: 'Select dates' },
```

Reply:
Are we default sentence case in most of the app?